packages/flutter/test/cupertino/route_test.dart

```diff
@@ -328,4 +328,110 @@ void main() {
     expect(find.text('route'), findsOneWidget);
     expect(find.text('push'), findsNothing);
   });
+
+  testWidgets('Fullscreen route animates correct transform values over time', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      CupertinoApp(
+        home: Container(
```

Comment:
I wonder if you really need this Container

Reply:
Tested without and the test passes so I removed it now.